help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,182 @@
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>* To activate data, you need the **[!UICONTROL View destinations]**, **[!UICONTROL Activate destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+Once the destination has been created, you will be shown the audience activation flow. The following walks through how to activate audiences using the Real-Time destination.
+
+#### Step 1: Select your new destination and click Next.
+
+![select the destination to activate audiences too](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-select-destination.png)
+
+#### Step 2: Select any audiences you want to activate, then click Next.
+
+![select the audiences to activate](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-select-audience.png)
+
+### Map attributes and identities {#map}
+
+The next step is mapping source identifiers to the Magnite device_id identifier.
+
+![map desired data fields to the device_id field](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-field-mapping.png)
+
+**Real-Time destination notes**:
+
+- You may have multiple source attributes/namespaces that could represent a Device ID, so map as many as you need.
+
+- A new mapping can be added using the Add new mapping button.
+
+In this example using the Real-Time destination, we're mapping any rows
+that contain a generic deviceId source identifier to the Magnite
+device_id target field. When you\'re done, click Next.
+
+![Be sure to set mapping ids to all activated audiences, or set NONE if no mapping id is present](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-mappingid.png)
+
+You must now configure a Start date (mandatory), End date (optional),
+and a Mapping ID for each audience.
+
+**Mapping ID notes:**
+
+- The Mapping ID field should be used when an audience has a pre-existing Segment ID previously known to Magnite.
```

Comment:
Please avoid the passive voice in docs.

Use the `**[!UICONTROL Mapping ID]**` field  when an audience has a pre-existing Segment ID previously known to Magnite.

Reply:
Updated